samigo/samigo-app/src/webapp/jsf/delivery/item/deliverFillInNumeric.jsp

```diff
@@ -153,5 +152,50 @@ window.onload = function() {
 	      });
 	});
 };
+
+includeWebjarLibrary('mathjs');
+var finFormatError = '<h:outputText value="#{deliveryMessages.fin_invalid_characters_error}" escape="false"/>';
+
+$( document ).ready(function() {
+	$('.fillInNumericInput').each(function() {
+		$(this).change(function(){
+			if(!$(this).val()){

```

Comment:
Don't need to wrap 'this' in a jquery object, just use this.value to get the input value.

Reply:
Please take a look at my most recent commit, thanks again!